docs/tutorials/using_apis/mesh/quick_start_build.md

```diff
@@ -74,3 +74,47 @@ Then you may optionally choose to select the non-routing mode for those networks
 |Mesh router (default) | `thread_router` | `MESH_DEVICE_TYPE_THREAD_ROUTER` |
 |Non-routing device | `thread_end_device` | `MESH_DEVICE_TYPE_THREAD_SLEEPY_END_DEVICE` |
 
+##### Configuration parameters for 6LoWPAN-ND and Thread
```

Comment:
We have a PR to move configuration content to the configuration section: https://github.com/ARMmbed/Handbook/pull/366

Could you please update that PR with this content in the format of the other sections?

Reply:
That might be ok, though I like the current format, for me it is more readable.